src/pages/AddJob.jsx

```diff
@@ -42,6 +43,73 @@ export default function AddJob() {
     }
   );
 
+  useEffect(() => {
+    let hasdraft = window.localStorage.getItem("draft");
+    if (hasdraft !== null && toastid.length === 0) {
+      toastid.push(1);
+      toast(
+        (t) => (
+          <div className="flex flex-row justify-between items-center w-[350px] p-4 rounded-lg shadow-md">
+            <div className="w-1/2">
+              <p>Load draft?</p>
+            </div>
+            <div className="flex flex-row w-1/2 justify-between h-full">
+              <div>
+                <button
+                  onClick={() => {
+                    loadDraft();
+                    setHasLoad(true);
+                    toast.dismiss(t.id);
+                  }}
+                  className="btn btn-primary btn-sm"
+                >
+                  Load
+                </button>
+              </div>
+              <div>
+                <button
+                  onClick={() => toast.dismiss(t.id)}
+                  className="btn btn-outline btn-primary btn-sm"
+                >
+                  Dismiss
+                </button>
+              </div>
+            </div>
+          </div>
+        ),
+        { duration: 60000 }
+      );
+    }
+  }, []);
+
+  const saveDraft = () => {
+    window.localStorage.setItem("draft", JSON.stringify(getValues()));
+    toast.success("Draft saved!");
+    navigate("/");
```

Comment:
not needed. just save draft and nothing happens

Reply:
Do you mean no navigate?